packages/next/build/utils.ts

```diff
@@ -71,16 +67,17 @@ export function printTreeView(
 
       const pageInfo = pageInfos.get(item)
 
+      const renderTarget = item.startsWith('/_')
+        ? ' '
+        : pageInfo && pageInfo.static
+        ? chalk.bold('⚡')
+        : serverless
+        ? 'λ'
+        : 'σ'
+      const maxWidth = Math.max(...[' ', 'λ', 'σ', '⚡'].map(stringWidth))
```

Comment:
Since these are static values, can't we hardcode the width?

Reply:
No, `string-width` is purely a function of the string input as far as I can tell from digging through the code.